spectator/meter/counter.go

```diff
@@ -8,54 +13,39 @@ package spectator
 //
 // https://netflix.github.io/spectator/en/latest/intro/counter/
 type Counter struct {
-	count uint64
-	// Pointers need to be after counters to ensure 64-bit alignment. See
-	// note in atomicnum.go
-	id    *Id
+	id              *Id
+	writer          writer.Writer
+	meterTypeSymbol rune
 }
 
 // NewCounter generates a new counter, using the provided meter identifier.
-func NewCounter(id *Id) *Counter {
-	return &Counter{0, id}
+func NewCounter(id *Id, writer writer.Writer) *Counter {
```

Comment:
Nice, I think this is a reasonably clean way go about it - passing it into the New for each meter type.